### PR TITLE
Maya/158557 phase out css modules

### PR DESCRIPTION
**Pull request by mmmmmmaya**
### Summary
- **What:** Phase out all remaining SCSS in the app
- **Why:** This is legacy code, and having two systems of styling components can cause problems. It is confusing for new developers who are trying to understand our conventions. It makes it unclear whether styles are coming from scss style classes or from styled components. It adds bloat to our code base in the form of additional dependencies. It makes refactoring certain components more challenging.
- **Ok, but why now, though:** I was working on a component that was partially styled with scss and partially styled with styled components and I decided I was done with it.
- **Ticket:** [[sc-158557]](https://app.shortcut.com/genepi/story/158557)
- **Env:** https://maya-cssmodules-frontend.dev.czgenepi.org/

### Demos
Should be no visual changes.

### Notes
I did my best to directly port over scss into styled components. I didn't try to make any smart changes or optimize in any way.

### Checklist
- [x] I merged latest `trunk`
- [x] I manually verified the change
- [x] I added labels to my PR
- [ ] I either asked design for a review or hid my changes behind a feature flag
- [ ] I tested in multiple browsers
- [ ] I added relevant unit tests
- [ ] I have notified others of changes they need to make locally (migrations, jobs, package updates, etc)

### CODE REVIEW

**Comment by vincent-czi** [comment]
None of the changes in package.json seem to involve anything that would actually convert `scss` to `css`: do you know where that happens? Is that part of webpack or the React build process or something else? //looking into it...// Okay, looks like it's based around the `"sass": "^1.38.2"` package. Can we remove that at this point, or is there something else still depending on that?

**Review by lvreynoso — APPROVED** [review]

File: src/frontend/src/components/NavBar/index.tsx [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
 import Link from "next/link";
 import React from "react";
 import { useUserInfo } from "src/common/queries/auth";
 import { ROUTES } from "src/common/routes";
 import RightNav from "./components/RightNav";
-import style from "./index.module.scss";
-import { Logo, LogoAnchor } from "./style";
+import { LeftNav, Logo, LogoAnchor, NavBar, NavOrg, Separator } from "./style";
 
 // (thuang): Please make sure this value is in sync with what we have in css
```
Comment (vincent-czi) [inline src/frontend/src/components/NavBar/index.tsx:8]: Neither here nor there about your work, but I think this comment is out of date and no longer matters. Checking the code, appears that `NAV_BAR_HEIGHT_PX` only shows up in one other file, where it's being imported from here.

File: src/frontend/src/views/Data/index.tsx [hunk 2]
```diff
@@ -128,28 +134,20 @@ const Data: FunctionComponent = () => {
...
       <Link href={category.to} key={category.text} passHref>
         <a href="passHref">
-          <Menu.Item className={style.menuItem}>
-            <div className={style.category}>
-              <div
-                className={cx(style.title, focusStyle)}
+          <StyledMenuItem>
+            <Category>
+              <CategoryTitle
+                isActive={router.asPath === category.to}
```
Comment (vincent-czi) [inline src/frontend/src/views/Data/index.tsx:143]: ahhh, this feels much better to read

**Review by vincent-czi — APPROVED** [review]
Looks good! I didn't do a super careful review because I didn't cross-reference all the deleted items with the newly added style components, but everything I read looked great.

Appreciate you doing a thankless but very useful clean-up task!

My one caveat is that it would be great if @Janeece could check out the rdev and give it a quick once-over since this is such a major way the styling internals are changing. I expect absolutely nothing will be a problem, but it would be nice to get her verification (if you haven't already).

**Comment by Janeece** [comment]
@mmmmmmaya RE: Vince's suggestion that I take a look, overall it seems like the styling was maintained. I found a few oddities not found on prod that I'm not sure are related:
Screen Shot 2022-05-20 at 10 12 44 AM
Screen Shot 2022-05-20 at 10 12 26 AM
Screen Shot 2022-05-20 at 10 12 02 AM

**Pull request merged**